src/GregClient/Requests/HeaderVersionDownload.cs

```diff
@@ -0,0 +1,51 @@
+﻿using RestSharp;
+
+namespace Greg.Requests
+{
+    public class HeaderVersionDownload : PackageReferenceRequest
```

Comment:
please add /// summary comments to all public classes and members where it makes sense - I know this repo doesn't follow this rule, but it should - it's set out in dynamo's coding standards:
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards

Reply:
Added a summary to the class. The public properties here are not really meant to be used from client code. They should probably be protected.

Reply:
Just noticed this, if that's better please make them internal even that may not follow the previous code pattern. Appreciate the thoughts put on!

Reply:
@QilongTang Unfortunately, that would require to change the accessibility of those properties in the base class and other requests too. Because it is technically a breaking change I'd rather not do it here.

Reply:
Make sense!